common/tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
-  "extends": "@tsconfig/node18/tsconfig.json",
+  "extends": "@tsconfig/node20/tsconfig.json",
   "compilerOptions": {
     "rootDir": ".",
+    "moduleResolution": "node",
+    "module": "esnext",
```

Comment:
`prismicio/types` struggles otherwise, they [do not support other moduleResolution types.](https://community.prismic.io/t/typescript-error-regarding-prismicio-client-after-switching-module-resolution-in-typescript/13154/5)